frontend/frames/settings.py

```diff
@@ -83,6 +93,46 @@ def __init__(self, window, base, **kwargs):
             command=self.close
         )
         self.close_button.grid(row=3, column=0)
+        self.insert_config_data(self.config)
+
+    def insert_config_data(self, config):
+        for fieldname, value in config.items():
+            if fieldname == "name":
+                self.name.set(value)
+
+    def save_config_data(self):
+        config_dir = os.path.dirname(os.path.abspath(__name__))
+        config_file = "alinka.ini"
+        file = open(os.path.join(config_dir, config_file), 'w')
+        config = ConfigParser()
+        config.add_section('Center')
+        config.set('Center', 'name', self.name.get())
+        config.write(file)
+        self.name_accept_button['state'] = 'disabled'
+
+    def track_changes(self, event, *args):
+        '''track if name in entry has been changed'''
+        if 'name' in self.config.items():
+            if self.name.get() == self.config['name']:
+                self.name_accept_button.config(state='disabled')
+            else:
+                self.name_accept_button.config(state='active')
+
+    def read_config(self):
+        init = {}
+        directory = os.path.dirname(os.path.abspath(__name__))
+        init_file = os.path.join(directory, "alinka.ini")
+        config = ConfigParser()
+        config.read(init_file)
+        sections = config.sections()
+        if not sections:
```

Comment:
this if does not make sense, as if `selection will be falsy, the whole `for` loop below will not run and `init = {}` will be returned (as here You have `return {}`).

More, if You have `return` in `if` statement the `else` clause doesn't make sense as these 2 sniipets will do exactly the same thing:
```
def f(a):
    if a:
        return 'a'
    else:
        return 'b'
```
and
```
def f(a):
    if a:
        return 'a'
    return 'b'
```